DATAFLOW.md

```diff
@@ -1,28 +1,27 @@
 # Enterprise Knowledge Solution (EKS) Dataflow
 
 1. **Upload documents and trigger document processing workflow**
-   An employee (IT) manually uploads various document types (Excel, PDF, Word, Text, Outlook, .xlsx, .pdf, .docx) or a scheduled Cron Job triggers the upload. The uploaded files are stored in a designated "input" bucket within Google Cloud Storage.
-
-   The upload to the "input" bucket triggers a Document Processing Workflow managed by Cloud Composer.
-
+   An employee (IT) manually uploads various document types (Excel, PDF, Word, Text, Outlook) or a scheduled Cron Job triggers the upload. The uploaded files are stored in a designated "input" bucket within Google Cloud Storage.
+   
+   The upload to the "input" bucket triggers a Document Processing Workflow managed by Cloud Composer.  
 1. **Validate the documents:**
    Documents that fail validation are redirected to a separate "reject" bucket within Google Cloud Storage for further review and handling.
 1. **Initiate Processing Jobs:**
    The workflow initiates specific processing jobs based on the document type:
-   `PDF Classifier (Cloud Run -> Doc AI Custom Classifier):` Identify PDF documents that contain forms.
-   `Forms Processor (Cloud Run -> Doc AI Form Parser):` Processes form documents(PDF).
-   `Excel, Outlook Processor (Cloud Run):` Processes Excel and Outlook files.
```

Comment:
Names of jobs here don't match the names of Jobs that we create, so it's hard to understand which is referring to which